### DISCUSSION TITLE
FullStack - Tempo de duração da autenticação por Cookie

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#637] [Tira Dúvidas] [answered]
Pessoal uma dúvida, (não sei se fala no curso e ainda não cheguei lá), mas tem como configurar o "tempo" de validade do Cookie ou ele é justamente para não ter esse "tempo" ao contrário de um JWT por exemplo.
Digo porque aqui em meus testes enquanto o cookie existir está autenticando...

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
https://github.com/balta-io/forum/discussions/594
